src/Symfony/Component/Validator/Constraints/ExpressionValidator.php

```diff
@@ -38,17 +38,22 @@ class ExpressionValidator extends ConstraintValidator
     private $expressionLanguage;
 
     /**
-     * @param PropertyAccessorInterface|null $propertyAccessor Optional as of Symfony 2.5
+     * @param PropertyAccessorInterface|null $propertyAccessor   Optional as of Symfony 2.5
+     * @param ExpressionLanguage             $expressionLanguage
      *
      * @throws UnexpectedTypeException If the property accessor is invalid
      */
-    public function __construct($propertyAccessor = null)
+    public function __construct($propertyAccessor = null, $expressionLanguage = null)
```

Comment:
That's what I wanted to ask as well. I just copied the lines for PropertyAccess but personally I'd use type hints.